### ISSUE TITLE
How to change MathJax Delimiters?

### ISSUE THREAD

**Issue by apelliciari** [KosmosisDire/obsidian-webpage-export#417]
Hi,

I have issues with $$ because I use $$$<string>$$$ for a lot of placeholders

This messes up the export because of the conflict of the delimiter with Mathjax?

**Is there a way to change the MathJax delimiters to other characters? Even forking if needed.**

I've tried to add to a file called export.html this:

```
<style>
  <script type="text/x-mathjax-config">
    MathJax.Hub.Config({
      tex2jax: {
        inlineMath: [['$','$'], ['£','£']],
        displayMath: [['$$','$$'], ['££','££']]
      }
    });
    </script>
    <script type="text/javascript" async
      src="https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS_CHTML">
    </script>
</style>
```

and added this in the custom head content, but it doesn't seem to work.

Thanks!

**Comment by KosmosisDire** [comment]
The plugin actually renders mathjax directly into the html to avoid including mathjax JavaScript. Is there a way to change the mathjax delimiters in obsidian?

**Comment by KosmosisDire** [comment]
No probably not, this is not something that most people would have as an issue. I am also not 100% I understand why the mathjax isn't working for you, can you maybe explain a bit more in detail how I could reproduce the issue?

**Comment by apelliciari** [comment]
Yes,
essentiallly I use obsidian to define technical documentation for marketing purpose.
So I have a lot of notes called
$$$currency$$$, $$$list_type$$$
these are "variables" that must be replaced by the developer with real values.
I can't currently change the symbol used.
So I don't really need mathjax, I need to turn it off or escaping it to avoid conflicts or messed up output. I was wondering if this is possible, and since I didn't found any options in obsidian or in the plugin, I thought that changing delimiters was a neat solution.

**Comment by KosmosisDire** [comment]
If you're comfortable modifying the code it should be pretty easy to remove the mathjax rendering from the plugin, I'll also make sure the plugin does not respond to $$$ and only responds to $ and $$

**Comment by apelliciari** [comment]
> If you're comfortable modifying the code it should be pretty easy to remove the mathjax rendering from the plugin, I'll also make sure the plugin does not respond to $$$ and only responds to $ and $$ in the future

yes honestly I tried but I couldn't find the exact point, also I'm not familiar with obsidian debugging so I'm not sure I can be successful on that.
Thanks anyway for the help, now I'm using the "code" workaround, i.e. surrounding the $$$ with ` ` so mathjax is not applied. Not the best solution but so it's not blocking.
